docs/csharp/language-reference/keywords/double.md

```diff
@@ -14,7 +14,7 @@ The `double` keyword signifies a simple type that stores 64-bit floating-point v
 
 |Type|Approximate range|Precision|.NET type|
 |----------|-----------------------|---------------|-------------------------|
-|`double`|±5.0 × 10<sup>−324</sup> to ±1.7 × 10<sup>308</sup>|15-16 digits|<xref:System.Double?displayProperty=nameWithType>|
```

Comment:
I think this should be explicit that it is approx 15-17 digits (same for float on 6-9 digits). There may be some values that aren't "exact" even for 15 digit inputs (depending on the exact input, its magnitude, etc).